.globality/build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "python-library",
+  "params": {
+    "name": "sklearn-hierarchical-classification",
+    "import-name": "sklearn_hierarchical",
```

Comment:
Is this another situation we want to deal with re: naming conventions? @moshez

Reply:
Yes. I added that to the same ticket :)